### PR TITLE
Custom input for notebooks with fewer test cases

### LINKED ISSUE #252: Custom code for notebook submissions
**Is your feature request related to a problem? Please describe.**
It would be helpful for students to write custom test or grading code to avoid many submissions.

**Describe the solution you'd like**
Add a custom code tool for notebooks as it is done for multilang

### PR DESCRIPTION
**Pull request by crdgonzalezca**
# Description

As in multilang the custom input is employed, here the students will be able to select up to 3 tests from the notebook and test the notebook without doing a new submission. This is supposed to leverage the server load as some submissions might be test through several tests.
 
Fixes #252 

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# How Has This Been Tested?

It was tested locally and everything looks perfect.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by Heldeg — CHANGES_REQUESTED** [review]
Seem right. I am only concerned with the comment I left below; the other is just a suggestion.

File: inginious/frontend/plugins/multilang/problems/custom_input_notebook.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+from inginious.frontend.parsable_text import ParsableText
+
+
+def custom_input_notebook(client):
+    """
+    This function returns a CustomInputNotebookManager in charge of running the selected tests by the student to run.
+    This is instead of doing a new submission whenever the student wants to check the result of some few tests.
+    :param client:
+    :return: CustomInputNotebookManager
+    """
```
[the remark this answers is not in the document]
Comment (crdgonzalezca) [reply inginious/frontend/plugins/multilang/problems/custom_input_notebook.py:16]: Can you please check this again? as I tested this on Firefox but it worked correctly.

[the remark this answers is not in the document]
Comment (AndresFelipeCastellanos) [reply inginious/frontend/plugins/multilang/problems/custom_input_notebook.py:16]: I tried to do the same with the notebook example in the test course and in another notebook task, but all the tests run regardless of which ones are selected.

[the remark this answers is not in the document]
Comment (AndresFelipeCastellanos) [reply inginious/frontend/plugins/multilang/problems/custom_input_notebook.py:16]: It's already working properly. Just need to update the changes made to the containers repository.

**Review by AndresFelipeCastellanos — APPROVED** [review]

**Pull request merged**